io-engine/src/subsys/config/opts.rs

```diff
@@ -765,6 +784,7 @@ impl From<&IoBufOpts> for spdk_iobuf_opts {
                 large_pool_count: o.large_pool_count,
                 small_bufsize: o.small_bufsize,
                 large_bufsize: o.large_bufsize,
+                enable_numa: 0,
```

Comment:
```suggestion
                enable_numa: o.enable_numa,
```

Reply:
We should add it to the `IoBufOpts` and make it tweakable from env, since this seems like it could have perf impacts for numa systems?